api/src/main/java/org/pmiops/workbench/firecloud/FireCloudServiceImpl.java

```diff
@@ -152,6 +150,12 @@ private void checkAndAddRegistered(FirecloudWorkspaceIngest workspaceIngest) {
     workspaceIngest.setAuthorizationDomain(ImmutableList.of(registeredDomain));
   }
 
+  @Override
+  @VisibleForTesting
+  public String getBasePath() {
```

Comment:
nit: maybe rename this to `getFirecloudApiBasePath()` so it's more obviously distinct from the service's object model.